.ci/pitest-suppressions/pitest-javadoc-suppressions.xml

```diff
@@ -0,0 +1,1973 @@
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>TagParser.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.TagParser</mutatedClass>
+    <mutatedMethod>parseTags</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.InlineConstantMutator</mutator>
+    <description>Substituted 0 with 1</description>
+    <lineContent>Point position = findChar(text, &apos;&lt;&apos;, new Point(0, 0));</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>JavadocStyleCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck</mutatedClass>
+    <mutatedMethod>trimTail</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.RemoveConditionalMutator_EQUAL_IF</mutator>
+    <description>removed conditional - replaced equality check with true</description>
+    <lineContent>else if (index &gt; 0 &amp;&amp; builder.charAt(index) == &apos;/&apos;</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>SummaryJavadocCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck</mutatedClass>
+    <mutatedMethod>isTextPresentInsideHtmlTag</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.InlineConstantMutator</mutator>
+    <description>Substituted 0 with 1</description>
+    <lineContent>isTextPresentInsideHtmlTag = !nestedChild.getText().isBlank();</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>JavadocStyleCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck</mutatedClass>
+    <mutatedMethod>trimTail</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.RemoveConditionalMutator_EQUAL_ELSE</mutator>
+    <description>removed conditional - replaced equality check with false</description>
+    <lineContent>while (builder.charAt(index - 1) == &apos;*&apos;) {</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>MissingJavadocMethodCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck</mutatedClass>
+    <mutatedMethod>isContentsAllowMissingJavadoc</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.RemoveConditionalMutator_EQUAL_IF</mutator>
+    <description>removed conditional - replaced equality check with true</description>
+    <lineContent>|| ast.getType() == TokenTypes.COMPACT_CTOR_DEF)</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>WriteTagCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.javadoc.WriteTagCheck</mutatedClass>
+    <mutatedMethod>logTag</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.InlineConstantMutator</mutator>
+    <description>Substituted 2 with 3</description>
```

Comment:
`InlineConstantMutator` could be good for mutating token values in checks, to make us rethink all tokens and related logic in certain checks, methods, etc. But - this mutator is causing a large number of violations that may require a lot of work to create all test cases to cover.

@rnveach @romani @pbludov @baratali @Vyom-Yadav please share your thoughts on the value of this mutator.

Link to docs: https://pitest.org/quickstart/mutators/#INLINE_CONSTS

Link to source: https://github.com/hcoles/pitest/blob/master/pitest/src/main/java/org/pitest/mutationtest/engine/gregor/mutators/InlineConstantMutator.java

Reply:
Given the number of violations and the degree of tediousness v/s effectiveness, I would like to vote in favor of not using this mutator.